scss/bitstyles/ui/forms.stories.mdx

```diff
@@ -80,16 +80,16 @@ The `aria-describedby` attribute accepts a space-separated list of IDs, so if an
   <Story name="Login form with errors">
     {`
       <div class="a-content a-content--s">
-        <form class="a-card@m u-padding-xl@m" method="POST" action="">
-          <div class="u-padding-l-y u-margin-neg-m u-margin-neg-xl@m u-margin-l-bottom@m a-flash a-flash--warning" role="alert">
-            <div class="a-content u-flex u-items-center">
+        <form class="a-card-l@m u-bg--white" method="POST" action="">
+          <div class="a-card-l__header@m a-flash a-flash--warning" role="alert">
```

Comment:
@hannahbit this line is much simpler now! (or will be, once this is released) I avoided adding to the increasingly-long list of utility classes, and made this a part of the `a-card` component